Diverology/Diverology/Source/Presentation/ListView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ListView.swift
+//  Diverology
+//
+//  Created by jeongminji on 4/15/25.
+//
+
+import SwiftUI
+
+struct ListView: View {
+    
+    @State var inputText: String = ""
+    @State private var selectedFilter: TimeFilter = .all
+    
+    let names = (1...30).map { "Diver \($0)" }
+    
+    let spacing: CGFloat = 20
+    let columns: Int = 3
+
+    var body: some View {
+        GeometryReader { geometry in
+            let totalSpacing = spacing * CGFloat(columns - 1)
+            let availableWidth = geometry.size.width - (20 * 2) - totalSpacing
+            let itemWidth = availableWidth / CGFloat(columns)
+            
+            ZStack(alignment: .top) {
+                BackgroundImageView()
+                
+                VStack(spacing: 20) {
+                    CustomTextField(
+                        text: $inputText,
+                        placeholder: "검색어를 입력해주세요.",
+                        image: Image(systemName: "magnifyingglass")
+                    )
+                    .padding(.horizontal, 20)
+                    
+                    HStack(spacing: 10) {
+                        ForEach(TimeFilter.allCases, id: \.self) { filter in
+                            FilterButton(
+                                title: filter.rawValue,
+                                isSelected: selectedFilter == filter
+                            ) {
+                                selectedFilter = filter
+                            }
+                        }
+                    }
+                    .padding(.horizontal, 20)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    
+                    ScrollView(showsIndicators: false) {
+                        LazyVStack(spacing: spacing) {
```

Comment:
```suggestion
                    .frame(maxWidth: .infinity, alignment: .leading)
                    
   		    // MARK: 프로필 스크롤
                    ScrollView(showsIndicators: false) {
                        LazyVStack(spacing: spacing) {
```

으로 View들을 MARK 부분으로 나눠두면 읽기도 편하고 추후에 수정도 편할 것 같아요!

Reply:
디테일 하시군요!! MARK 나눠두면 다른 분들이 코드 읽기도 훨씬 펼할 것 같습니다! 좋습니다👏🏻